schemas/ajv.absolutePath.js

```diff
@@ -40,8 +38,12 @@ module.exports = ajv =>
 					];
 					passes = false;
 				}
-
-				if (!isCorrectAbsoluteOrRelativePath) {
+				// ?:[A-Za-z]:\\ - Windows absolute path
+				// \\\\ - Windows network absolute path
+				// \/ - Unix-like OS absolute path
+				const isCorrectAbsolutePath =
+					expected === /^(?:[A-Za-z]:(\\|\/)|\\\\|\/)/.test(data);
```

Comment:
```suggestion
					expected === /^(?:[A-Za-z]:\\|\\\\|\/)/.test(data);
```

correct windows paths must use `\` and not `/`.

Reply:
Made changes

Reply:
@sokra Windows supports `c:/foo/bar` https://nodejs.org/api/path.html#path_path_isabsolute_path